BUILD.bazel

```diff
@@ -12,7 +12,7 @@ config_setting(
 generate_header(
     name = "IexConfig.h",
     substitutions = {
-        "@IEX_INTERNAL_NAMESPACE@": "Iex_3_0",
+        "@IEX_INTERNAL_NAMESPACE@": "Iex_3_1",
```

Comment:
I'm wondering if this will be a problem when OpenEXR hits actual version 3.1? Would it make sense to instead call this Iex_3_0_1? A minor bump isn't supposed to cause an incompatibility at the interface level ~ did we inadvertently do something that broke compatibility from the 3.0 interface series?

Reply:
Other than this point, the rest of the diff LGTM.

Reply:
Ups... did no may change was wrong - should be Iex_3_0 - changed this